src/App.tsx

```diff
@@ -1,51 +1,54 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
-
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+import {
+  Link,
+  Route,
+  Routes,
+  Navigate,
+  NavLink,
+} from 'react-router-dom';
+import classNames from 'classnames';
+import { TabPage } from './pages/TabPage';
 
 export const App = () => (
   <>
-    {/* Also requires <html class="has-navbar-fixed-top"> */}
     <nav
-      className="navbar is-light is-fixed-top is-mobile has-shadow"
+      className="navbar is-light is-fixed-top"
       data-cy="Nav"
     >
       <div className="container">
         <div className="navbar-brand">
-          <a href="/" className="navbar-item is-active">Home</a>
-          <a href="/tabs" className="navbar-item">Tabs</a>
+          <Link
+            to="/"
+            className="navbar-item"
+          >
+            Home
+          </Link>
+
+          <NavLink
+            to="tabs"
+            className={({ isActive }) => classNames(
+              'navbar-item', { 'is-active': isActive },
+            )}
+          >
+            Tabs
+          </NavLink>
```

Comment:
Would be great to create a separate component for the navigation. 
Also, your navigation links should be the same. Now the Home link doesn't even have is-active class